### ISSUE TITLE
UI controls not accessible through keyboard under metrics in application live view plugin

### ISSUE THREAD

**Issue by claudiahub** [backstage/backstage#11295]
## Expected Behavior

UI controls should be accessible through keyboard under metrics in application live view plugin.

## Actual Behavior

1. Navigate to `Home` on the left navigation. Make sure `Components` is selected on menu below the main heading.
2. Select `all` filter and in the table navigate to any application under name column with application live view plugin available (see screenshot below).
3. In the next page that opens up for selected appplication, please navigate to `Runtime resources` tab.
4. Navigate to `Resources` table and select the name of the application. A new page opens up.
5. Navigate to `Pods` table and select a link under name column.
6. In the new page that opens up, navigate to `Live view` section.
7. Select `Metrics` item from the menu item.
8. Verify whether multiple comboboxes and buttons (e.g. Add metric) are accessible through keyboard.

<img width="1622" alt="image" src="https://user-images.githubusercontent.com/54085737/166665457-cfa3048f-a86d-4e6b-a91d-3b0839ffc2d5.png">

## Context

Ensure all interactive elements can receive focus and are actionable via keyboard. For example, links use the ENTER key while buttons should allow ENTER and SPACE key.

Involved component(s):

- Select from '@backstage/core-components'
- Button from '@backstage/core-components'

User Impact: 
When any UI component are not accessible by using keyboard then user who rely on keyboard are facing difficulty and not able to perform any action and are completely block to perform any functionality on that control element.

A11y Conformance Level: A

WCAG Success Criteria: 2.1.1

## Environment

Browser Information: 
Google Chrome v100.0.4896.127
Firefox 99.0.1
Safari 15.4

**Comment by freben** [comment]
Thanks for making all of these, by the way! 🥳

**Comment by freben** [comment]
@claudiahub We are not sure that we recognise this view at all - could it be an internal plugin that you have?

**Comment by claudiahub** [comment]
@freben this is, as you mentioned, an internal plugin but the components in this view are located on @backstage/core-components so I guess we should address there this accessibility issue. What do you think ?

**Comment by claudiahub** [comment]
@freben a Pr I did recently ➡️https://github.com/backstage/backstage/pull/11339, I guess this should close also this issue.

**Comment by claudiahub** [comment]
@freben Hi, I guess this can be closed.
It's on an internal plugin that #11339 fixed. 
Thanks!
